ansible/templates/kubeadm-config.yaml.j2

```diff
@@ -46,7 +46,7 @@ apiServer:
 {% endif %}
   extraArgs:
     authorization-mode: "Node,RBAC"
-    oidc-issuer-url: "https://dex.{{ lookup('env','ECK_BASE_DOMAIN') }}"
+    oidc-issuer-url: "{{ oidc_issuer_url }}"
     oidc-client-id: "kubelogin"
     oidc-username-claim: "email"
     oidc-groups-claim: "groups"
```

Comment:
I think it would make sense to expose all of these (or make it possible to add/override any of the extra args) but maybe that is for another PR. Anyway, it doesn't seem far fetched that you may want to use a different OIDC provider than dex (or configure dex differently) and then you may need different values here. :slightly_smiling_face:

Reply:
Good point! Will expose them as more config options.